app/views/layouts/application.html.erb

```diff
@@ -15,7 +15,7 @@
 
   <body>
     <div class="navbar-fixed">
-      <nav>
+      <nav class=<%= request.url.include?("dashboard") ? "nav-extended" : "" %>>
```

Comment:
This is good...it definitely works...but I wonder if it's getting a little hacky to be checking the url? Can't we just assign a variable in the controller to activate this if needed?